README.md

```diff
@@ -72,6 +74,8 @@ There is also a `deploy` command to deploy all Twinkle files live.
     ./sync.pl --deploy twinkle.js
     make deploy
 
+Note that for syncing to a custom wiki, you wil also need to specify the --lang and --family parameters too. For instance, to sync the files with `test.wmflabs.org` you should specify `--lang=test --family=wmflabs`. If you intend to use `make deploy` to deploy all the files at once, you may also need to modify the makefile to specify those parameters.
```

Comment:
Easiest way is probably to make your changes and then amend the previous commit with `git commit --amend`, then force pushing it to overwrite the old commit.